api-test.openfisca.fr/api-test.openfisca.fr.conf

```diff
@@ -0,0 +1,31 @@
+server {
+	listen 80;
+	server_name api-test.openfisca.fr;
+
+	location /.well-known {
+		root /tmp/renew-webroot;
+	}
+
+	location / {
+		return 302 http://fr.openfisca.org/api/v2$request_uri;
+	}
+}
+
+server {
+	listen 443 ssl;
+	server_name api-test.openfisca.fr;
+
+	ssl_certificate /etc/letsencrypt/live/api-test.openfisca.fr/fullchain.pem;
+	ssl_certificate_key /etc/letsencrypt/live/api-test.openfisca.fr/privkey.pem;
+
+	access_log /var/log/nginx/api-test.openfisca.fr-access.log;
+	error_log /var/log/nginx/api-test.openfisca.fr-error.log;
+
+	location / {
+		proxy_pass http://localhost:6000;
```

Comment:
As written on the issue : 

> Use a 302 to redirect api-test vers fr.openfisca.org/api

The traffic coming to `https://api-test.openfisca.fr` should also be redirected to `http://fr.openfisca.org/api/v2`.